lang/rustup/Makefile

```diff
@@ -0,0 +1,70 @@
+#
+# Copyright (C) 2020 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=rustup
+PKG_VERSION:=1.22.1
+PKG_RELEASE:=1
+PKG_LICENSE:=MIT
+
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_URL:=https://github.com/rust-lang/rustup.git
+PKG_SOURCE_DATE:=2020-07-18
+PKG_SOURCE_VERSION:=ac1a8c415b5c2886275ca92a3a8f3a4cd0587711
+PKG_HASH:=46c1cf800e48f43bf0b3e19bf1eb2ed414106e5ff2647cf7858286489454acf2
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+
+include $(INCLUDE_DIR)/host-build.mk
+include $(INCLUDE_DIR)/package.mk
+include ./rustc-triple.mk
+
+define Package/rustup
+    SECTION:=lang
+    CATEGORY:=Languages
+    TITLE:=Rust Programming Language Installer
+    URL:=https://www.rust-lang.org/
+endef
+
+define Package/rustup/description
+ rustup installs The Rust Programming Language from the official release
+ channels, enabling you to easily switch between stable, beta, and nightly
+ compilers and keep them updated. It makes cross-compiling simpler with binary
+ builds of the standard library for common platforms. And it runs on all
+ platforms Rust supports, including Windows.
```

Comment:
It might be more appropriate to put somethign here more relevant to rustup on openwrt?

Reply:
Thank you for the critique - Updated!